MeshGeoToolsLib/MeshNodesAlongSurface.h

```diff
@@ -39,7 +39,8 @@ class MeshNodesAlongSurface
      * @param mesh_nodes Nodes the search will be performed on.
      * @param sfc Along the GeoLib::Surface sfc the mesh nodes are searched.
      */
-    MeshNodesAlongSurface(MeshLib::Mesh const& mesh, GeoLib::Surface const& sfc, bool search_all_nodes = true);
+    MeshNodesAlongSurface(MeshLib::Mesh const& mesh, GeoLib::Surface const& sfc,
+                          double epsilon, bool search_all_nodes = true);
```

Comment:
Looks like the docs need some update. There's another default value here, but that's probably for another PR.